remove media_build patch due "rm -rf drivers/staging/media/atomisp" hack at LE/tools/mkpkg/mkpkg_pvr

diff --git a/backports/v4.10_sched_signal.patch b/backports/v4.10_sched_signal.patch
index af66823..67ed344 100644
--- a/backports/v4.10_sched_signal.patch
+++ b/backports/v4.10_sched_signal.patch
@@ -257,19 +257,6 @@ index add2edb..8eb0f49 100644
  #include <linux/slab.h>
  #include <linux/interrupt.h>
  
-diff --git a/drivers/staging/media/atomisp/pci/atomisp2/hmm/hmm_bo.c b/drivers/staging/media/atomisp/pci/atomisp2/hmm/hmm_bo.c
-index 79bd540..c9a61ba 100644
---- a/drivers/staging/media/atomisp/pci/atomisp2/hmm/hmm_bo.c
-+++ b/drivers/staging/media/atomisp/pci/atomisp2/hmm/hmm_bo.c
-@@ -34,7 +34,7 @@
- #include <linux/errno.h>
- #include <linux/io.h>
- #include <asm/current.h>
--#include <linux/sched/signal.h>
-+#include <linux/sched.h>
- #include <linux/file.h>
- 
- #include <asm/set_memory.h>
 diff --git a/include/media/v4l2-ioctl.h b/include/media/v4l2-ioctl.h
 index a7b3f7c..a63034b 100644
 --- a/include/media/v4l2-ioctl.h
-- 
2.14.1

